src/OrbitGl/Track.h

```diff
@@ -37,6 +37,7 @@ class Track : public orbit_gl::CaptureViewElement, public std::enable_shared_fro
     kGpuTrack,
     kSchedulerTrack,
     kAsyncTrack,
+    kMemoryTrack,
```

Comment:
Would you do me a favor and change this from `enum Type` to `enum class Type`. Since you are not technically touching this, feel free to dismiss. But I think its a fairly small change, that could be its own commit and would help a lot.

Reply:
I'm glad to do this. Thanks for the suggestion!